documentation/staging/content/userguide/managing-operators/using-helm.md

```diff
@@ -504,6 +505,20 @@ Example:
 elasticSearchPort: 9201
 ```
 
+##### `createLogStashConfigMap`
+Specifies whether a ConfigMap named `weblogic-operator-logstash-cm` should be created during `helm install`. 
+The ConfigMap contains Logstash pipeline configuration for the Logstash container running in the operator pod.
```

Comment:
contains Logstash pipeline configuration -> contains the Logstash pipeline configuration